packages/blade/src/components/Input/BaseInput/BaseInputTagSlot.native.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import type { BaseInputTagSlotProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import { Text } from '~components/Typography';
+
+const BaseInputTagSlot = ({
+  tags,
+  tagRows,
+  showAllTags,
+}: BaseInputTagSlotProps): React.ReactElement | null => {
+  if (!tags) {
+    return null;
+  }
+
+  const invisibleTagsCount = tags.length - 3;
+
+  if (tags.length <= 0) {
+    return null;
+  }
+
+  return (
+    <BaseBox
+      paddingLeft="spacing.4"
+      marginY="spacing.2"
+      justifyContent="flex-start"
+      display="flex"
+      flexDirection="column"
+      maxHeight="100px"
+      // Move to using gap instead of marginLeft on individual tags after RN upgrade
+      // gap="spacing.3"
```

Comment:
You can use gap now

Reply:
It's not working as expected for some reason 😕 I have kept the comment for now. Will take a look later.